knowledge-base/examples/chart-tooltip-series-item/main.vue

```diff
@@ -0,0 +1,160 @@
+          :visible="chartLayout.title.visible"
+          :align="chartLayout.title.align"
+          :position="chartLayout.title.position"
+          :text="title"
+        />
+        <ChartLegend :position="'bottom'" :orientation="'horizontal'" />
+        <ChartCategoryAxis>
+          <ChartCategoryAxisItem
+            :type="categoryAxisType"
+            :base-unit="'fit'"
+            :tooltip="{ visible: true, template: tooltipTemplate }"
+          >
+            <ChartCategoryAxisLabels
+              :rotation="'auto'"
+              :content="labelContent"
+              :visual="yAxisLabelContent"
+            />
+            <ChartCategoryAxisTitle
+              :text="category"
+              :visible="chartLayout.category.visible"
+            />
+          </ChartCategoryAxisItem>
+        </ChartCategoryAxis>
+        <ChartSeries>
+          <ChartSeriesItem
+            v-for="field in fields"
+            :key="field.Id + field.fieldName"
+            :type="type"
+            :data-items="datasetProcessed"
+            :field="field.fieldName"
+            :aggregate="field.aggregate"
+            :category-field="category"
+            :tooltip="{ visible: true, template: tooltipTemplate }"
+          />
+        </ChartSeries>
+      </Chart>
+      <Popup :offset="offset" :show="show" :animate="false">
+        {{ popupContent }}
+      </Popup>
+    </div>
+  </template>
+  
+  <script>
+  import {
+    Chart,
+    ChartTitle,
+    ChartSeries,
+    ChartSeriesItem,
+    ChartCategoryAxis,
+    ChartCategoryAxisLabels,
+    ChartCategoryAxisItem,
+    ChartCategoryAxisTitle,
+    CategoryAxisRangeLabels,
+    ChartLegend,
+  } from '@progress/kendo-vue-charts';
+  import { Popup } from '@progress/kendo-vue-popup';
+  import 'hammerjs';
+  
+  export default {
+    components: {
+      Chart,
+      ChartTitle,
+      ChartSeries,
+      ChartSeriesItem,
+      ChartCategoryAxis,
+      ChartCategoryAxisLabels,
+      ChartCategoryAxisItem,
+      ChartCategoryAxisTitle,
+      CategoryAxisRangeLabels,
+      ChartLegend,
+      Popup,
+    },
+    data: function () {
+      return {
+        popupContent: '',
+        show: false,
+        offset: { top: 150, left: 50 },
+        chartLayout: {
+          title: {
+            visible: true,
+            position: 'top',
+            align: 'center',
+          },
+          category: {
+            visible: true,
+          },
+        },
+        category: 'ApprovalStatus.Name',
+        categoryAxisType: 'category', // can be 'date'
+        title: 'Purchases',
+        fields: [
+          {
+            fieldName: 'Actual.TotalConsumption',
+            table: 'projects',
+            aggregate: 'sum',
+            text: 'Cost: Actual. Total',
+          },
+        ],
+        type: 'bar',
+        datasetProcessed: [
+          {
+            Id: 31289,
+            ApprovalStatus: {
+              Name: 'Approved by a very log whatnot label here sooo long',
```

Comment:
Make sure that the long message does not contain so many typos. E.g., you can change it to 
"Approved by a very long, loooong and even longer label shown inside a tooltip"